src/test/java/bogdanSandu/tests/Tema7/CURAHealthcare.java

```diff
@@ -0,0 +1,88 @@
+package bogdanSandu.tests.Tema7;
+
+import bogdanSandu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class CURAHealthcare {
+    static ChromeDriver driver = null;
+
+    @BeforeMethod
+    public void openCURAHealthPage() {
+        //open home page
+        driver = BrowserManager.createDriverAndGetPage();
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        driver.manage().window().maximize();
+        System.out.println("Am deschis CURA Healthcare Service page");
+    }
+
+    @Test
+    public void login() {
+        // go to login page
```

Comment:
This method and invalidlogin() method can be merged into one method if you use @DataProvider. Try this approach and let me know if you have any issues. Thanks!